### DISCUSSION TITLE
Intercepting route modal not working with base path

### DISCUSSION THREAD

**Discussion by Uy575** [vercel/next.js#52627] [App Router]
i created modal using intercepting route it was working fine on /modal but when basepath added it starts working as normal route is this a bug or any solution given by next?

**Comment by hyjoong** [comment]
I'm not sure exactly what specific case you're referring to. Did you apply the 'slot' using the '@' symbol successfully? Please refer to the https://github.com/vercel-labs/nextgram/tree/main/src/app/%40modal


Is the case you are talking about the same structure as the picture below?

<img width="184" alt="image" src="https://github.com/vercel/next.js/assets/70426440/b25aedc7-6c96-4fe1-adf4-29bec730412f">

**Reply by Uy575** [reply]
no i dont have any problem with structure and yes i created it succussfully and it was working fine,
here is brief explaination of my case:
i created a modal using intercepting route and parallel route it was working perfectly fine for me but when i added basepath a functionality given by next js that you can use both parallel and intercepting route to create modal was broken and they both start acting like individual behaviour.

**Reply by Uy575** [reply]
i identify the cause by my own and posted this issue on vercel platform, they marked it as a flag, now looking forward, untill then i forced to use custom modal, without basepath this feature is fine but with basepath its broken.
